manifests/manage/hg.pp

```diff
@@ -0,0 +1,18 @@
+# @summary Manage the Mercurial source code manager package
+#
+# @param package_name
+#   name of package to manage
+#
+# @param package_ensure
+#   ensure state of the package resource
+#
+# @example simple include
+#   include vcsrepo::manage::hg
+class vcsrepo::manage::hg (
+  Variant[String[1], Array[String[1]]] $package_name = 'mercurial',
```

Comment:
Almost there @jcpunk...
Looks like this is causing an error on rhel-9, failing with `Execution of '/bin/dnf -d 0 -e 1 -y install mercurial' returned 1: Error: Unable to find a match: mercurial`

Reply:
I'm tempted to drop `hg` as I did with `bzr` since I don't seem to be having much luck with the excludes...